packages/web-components/src/checkbox/checkbox.spec.ts

```diff
@@ -14,17 +14,21 @@ test.describe('Checkbox', () => {
 
     await expect(element).toHaveCount(1);
 
-    await page.setContent(/* html */ `
-        <fluent-checkbox shape="circular"></fluent-checkbox>
-    `);
-
-    await expect(element).toHaveJSProperty('shape', 'circular');
+    await test.step('should set the `shape` property to `circular`', async () => {
+      await page.setContent(/* html */ `
```

Comment:
Is it worth toggling the attribute to ensure it gets removed properly?

Reply:
It is! I went ahead and added an attribute toggle step to common attribute tests.